doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

```diff
@@ -1,6 +1,22 @@
 # Upgrade Notes
 
 ## 10.5.8
+- [Twig] Sending mails and Dataobject Text Layouts, which allow rendering user controlled twig templates are now executed in a sandbox with restrictive security policies for tags, filters, functions and son on.
```

Comment:
we need to add that information also to documentation it self... 

like [here](https://pimcore.com/docs/pimcore/current/Development_Documentation/Development_Tools_and_Details/Email_Framework/index.html) and [here](https://pimcore.com/docs/pimcore/current/Development_Documentation/Objects/Object_Classes/Layout_Elements/Dynamic_Text_Labels.html)

Reply:
done